packages/scrollable-video/src/components/video.js

```diff
@@ -93,6 +93,34 @@ function removeListenerFromMQList(mql, listener) {
   }
 }
 
+/**
+ *
+ * On iOS low power mode, autoplaying video is not allowed.
+ * Also, video cannot be played unless user interact with the device in such situation.
+ * This hack plays video on user first touch for hiding the playback button UI when it cannot be autoplayed.
+ */
+function playVideoWhenAutoplayIsDisabledOnTouch() {
+  if (typeof window === 'undefined') return
+  const playVideoWhenAutoplayIsDisabled = () => {
+    const videos = document.querySelectorAll('video')
```

Comment:
Questions 🙋 :
Won't it query other video elements (not scrollable video elements) as well?

Reply:
Thanks, I'll use classname as the selector instead.

Updated in https://github.com/twreporter/orangutan-monorepo/pull/110/commits/c89c2afadd2ee9c6e2004fd8b235a3de5a213ef8